src/sentry/monitors/endpoints/organization_monitors.py

```diff
@@ -74,7 +74,12 @@ def get(self, request: Request, organization: Organization) -> Response:
         environments = None
         if "environment" in filter_params:
             environments = filter_params["environment_objects"]
-            queryset = queryset.filter(monitorenvironment__environment__in=environments)
+            if request.GET.get("include_new"):
```

Comment:
This looks good to me. Are we okay with this parameter name @wedamija (from an API consistency perspective)

Reply:
ah yeah was going to ask about the param name

Reply:
Usually these are camel case to be consistent for FE. Should it be `includeNotStarted` or something? We should come up with a standard name for monitors that have no checkins and just use that

Reply:
I prefer `includeNew` from an endpoint perspective. I think as a name for the object it'd be more like "empty monitor", but logically the way its use is intended here it's `new`

Reply:
I am good with `includeNew`. Once we have docs on this we can include that there as well.